4788/src/main/cpp/Robot.cpp

```diff
@@ -3,60 +3,81 @@
 using namespace frc;
 using namespace wml;
 
+using hand = frc::XboxController::JoystickHand;
+
 double currentTimeStamp;
 double lastTimeStamp;
-double dt;
+double dt; //stands for delta time 
+
+//add other variables here
+double SparkSpeed;
+double TalonSpeed;
 
-// Robot Logiccd
+// Robot Logic
 void Robot::RobotInit() {
-	// Init the controllers
-	ControlMap::InitsmartControllerGroup(robotMap.contGroup);
+	
+	dt = frc::Timer::GetFPGATimestamp() - lastTimeStamp;
+	lastTimeStamp = frc::Timer::GetFPGATimestamp();
 
-	// Create wml drivetrain
-	drivetrain = new Drivetrain(robotMap.driveSystem.drivetrainConfig, robotMap.driveSystem.gainsVelocity);
+	//init controllers 
+	xbox = new frc::XboxController(0);
 
-	
-	// Zero Encoders
-	robotMap.driveSystem.drivetrain.GetConfig().leftDrive.encoder->ZeroEncoder();
-	robotMap.driveSystem.drivetrain.GetConfig().rightDrive.encoder->ZeroEncoder();
+	//Motor examples 
+	SparkMotor[0] = new frc::Spark(0);
+	TalonMotor = new wml::TalonSrx(1);
 
-	// Strategy controllers (Set default strategy for drivetrain to be Manual)
-	drivetrain->SetDefault(std::make_shared<DrivetrainManual>("Drivetrain Manual", *drivetrain, robotMap.contGroup));
-	drivetrain->StartLoop(100);
+	SparkMotor[0]->SetInverted(true);
+	TalonMotor->SetInverted(false);
 
-	// Inverts one side of our drivetrain
-	drivetrain->GetConfig().rightDrive.transmission->SetInverted(true);
-	drivetrain->GetConfig().leftDrive.transmission->SetInverted(false);
+	//registers the pressure sensor as a sensor to be logged to Network Tables
+	NTProvider::Register(&pressureSensor); 
 
-	// Register our systems to be called via strategy
-	StrategyController::Register(drivetrain);
-	NTProvider::Register(drivetrain);
 }
 
 void Robot::RobotPeriodic() {
-	currentTimeStamp = Timer::GetFPGATimestamp();
-	dt = currentTimeStamp - lastTimeStamp;
-
-	StrategyController::Update(dt);
-	NTProvider::Update();
-
-	lastTimeStamp = currentTimeStamp;
+	if (pressureSensor.GetScaled() < 80) {
```

Comment:
Pressure sensor probably won't be added this way. We generally use a pressure switch and only ever use the sensor to display the current pressure to shuffleboard. Because the pressure switch is a physical switch when pressure gets to a certain point. You can set this to on and leave it like that while the robot is enabled